modules/menu/menu.c

```diff
@@ -544,6 +545,40 @@ static int hold_prev_call(struct re_printf *pf, void *unused)
 	return call_hold(ua_prev_call(uag_cur()), true);
 }
 
+static int switch_audio_dev(struct re_printf *pf, void *arg)
+{
+	const struct cmd_arg *carg = arg;
+	static bool switch_aud_inprogress;
+	char *mod;
+	char *device;
+	int err = 0;
+
+	if (!switch_aud_inprogress && !carg->complete) {
+		statmode = STATMODE_OFF;
+		re_hprintf(pf, "\rPlease enter transfer target SIP uri:\n");
+	}
+
+	switch_aud_inprogress = true;
+
+	if (carg->complete) {
+
+		mod = strtok( carg->prm, ",");
+		device = strtok( NULL, ",");
```

Comment:
as Dmitrij correctly pointed out, strtok() is not thread-safe and should
not be used in baresip. In baresip we prefer to use re_regex, example:

```
re_regex(carg->prm, str_len(carg->prm), "[^,]+,[~]*", &pl1, &pl2);
```

and then convert the pl1/pl2 to string.